### ISSUE TITLE
Instrumenting non-strict file, this may cause unexpected behavior

### ISSUE THREAD

**Issue by ghoullier** [WarpJS/beta#16]
Latest `@warpjs/webpack-plugin` log the following error

```console
LOG from node_modules/@warpjs/webpack-plugin/lib/warpifyLoader src/api.js
<w> Instrumenting non-strict file, this may cause unexpected behavior: api.js
```

## api.js

```js
function hello() {
  'warp +server -client';

  return `Hello from Node`;
}

export { hello };
```

EcmaScript modules are automatically in strict mode [ref](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Strict_mode#Strict_mode_for_modules) [spec](https://tc39.es/ecma262/#sec-module-environment-records-getbindingvalue-n-s) so I dont understand the log message.

Note: my webpack config has not babel or typescript plugin to transpile my code.

**Comment by scajegou** [comment]
Indeed, this message is a false warning in the case of EcmaScript modules.
Thanks for your feedback

**Comment by jpl-scale** [comment]
Resolved in 1.1.5.
